math_notes/backend.py

```diff
@@ -145,11 +143,9 @@ def _ocr_request_button(
         latex_return = ["x"]
 
     # Remove the temporary directory for storing the canvas image.
-    try:
-        mydir = Path("./math_notes/temp_files/")
+    mydir = Path("./math_notes/temp_files/")
+    if os.path.isdir(mydir):
```

Comment:
You can do this stuff with pathlib too. Try to be consistent, and avoid going back and forth between pathlib and the older combo of `os` and `shutil`, where possible